import/popups/PromptPopupInterface.qml

```diff
@@ -14,6 +14,7 @@ import Sailfish.Silica 1.0
 UserPromptInterface {
     // inputs
     property string text
+    property string defaultValue
```

Comment:
This is an addition to the interface API.  Is this allowed, at this stage, or are these APIs supposed to be considered stable already?

Reply:
Just sumarising an internal discussion on this point (@atatarov, please correct me if I'm mischaracterising it): 
@llewelld: "What's OMP's position on this? Are we still able to adapt the API?"
@atatarov: "I think it's ok"

Reply:
It's ok to change such API at this stage.

Reply:
Marking as resolved.